frontend/src/components/DeviceShareContainer/ContactCardActions.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import {  Button, Chip, IconButton, Tooltip } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
+import { Icon } from '../Icon'
+
+export function ShareDetails({
+  scripting,
+  shared,
+}: {
+  scripting: boolean
+  shared: number
+}): JSX.Element {
+  const css = useStyles()
+  return (
+    <div className={css.contentDetail}>
+      {scripting && (
+        <Tooltip enterDelay={500} title="Sharing ability to execute scripts">
+          <IconButton className={css.icon}>
```

Comment:
Let's remove the <IconButton> component here since there isn't a click action. Thanks!